src/containers/CtypeCreate/CtypeCreate.tsx

```diff
@@ -83,18 +84,22 @@ class CTypeCreate extends React.Component<Props, State> {
           blockUi.updateMessage(
             `CTYPE stored on blockchain,\nnow registering CTYPE`
           )
-          const cTypeWrapper: ICType = {
-            cType,
+          const cTypeWrapper: CTypeMetadata = {
```

Comment:
I had to create a new type interface in order to match what is taken in the database.

I replicated the data after testing with my changes and then alongside what the demo client currently produces. I was able to get the information to match and not make so many changes to the backend. Unfortunately, there was a minor change due to a test and console log.

I have tried to match the state as seen above with `metadata`. Ideally, I would have liked to change a lot more. After attempting this I realised it was wasting too much time and felt unnecessary for the ticket.